packages/ui/src/assets/css/page.css

```
@@ -1,12 +1,11 @@
 .page {
   display: grid;
   grid-template-areas:
-    "breadcrumb"
-    "title"
-    "content"
-    "toolbar";
-  grid-template-columns: 100%;
-  grid-template-rows: auto 1fr auto;
+  "breadcrumb breadcrumb"
+  "title toolbar"
+  "content content";
```

Would it be more safe to display all in one column in mobile view?
```suggestion
  "breadcrumb"
  "title"
  "toolbar"
  "content";
```